csrc/tensor_view.cpp

```diff
@@ -240,69 +240,6 @@ std::string TensorView::toInlineString(int indent_size) const {
   return toString(indent_size);
 }
 
-void TensorView::convertRfactorToRootDomain() {
-  // For a given TensorView, does its domain (root / rfactor) contain any
-  // concrete sized extents?
-  auto is_concrete_tensor = [](TensorView* tv) {
-    for (auto id : tv->getMaybeRFactorDomain()) {
-      if (!id->extent()->isConstScalar()) {
-        return false;
-      }
-    }
-    return true;
-  };
-
-  // Create a new root domain and replacement TensorDomain.
-  // Given an rfactor domain, create a new IterDomain.
-  // Otherwise, clone the previous IterDomain
-  auto createReplacementDomain =
-      [this](const std::vector<Val*>& replacement_extents) {
-        TORCH_INTERNAL_ASSERT(
-            !replacement_extents.empty() &&
-            getMaybeRFactorDomain().size() == replacement_extents.size());
```

Comment:
I omitted the check that `replacement_extents` is not empty since it did not seem necessary.